### PR TITLE
{bio}[dummy/] Sambamba v0.6.6

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/Sambamba/Sambamba-0.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+sanity_check_paths = {
+    'files': ['sambamba_v%(version)s'],
+    'dirs': [],
+}
+
+modextrapaths = {
+    'PATH': "",
+}
+
+postinstallcmds = ['cd %(installdir)s && ln -s sambamba_v%(version)s sambamba',]
```
Comment (boegel) [inline easybuild/easyconfigs/s/Sambamba/Sambamba-0.6.6.eb]: @pescobar please move this up, above `sanity_check_paths`, and remove the trailing `,`

File: easybuild/easyconfigs/s/Sambamba/Sambamba-0.6.6.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ written in the D programming language, for working with SAM and BAM files. Current functionality 
+ is an important subset of samtools functionality, including view, index, sort, markdup, and depth"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/lomereiter/sambamba/releases/download/v%(version)s/']
+sources = ['%(namelower)s_v%(version)s_linux.tar.bz2']
+
+sanity_check_paths = {
+    'files': ['sambamba_v%(version)s'],
```
Comment (boegel) [inline easybuild/easyconfigs/s/Sambamba/Sambamba-0.6.6.eb]: also check for the symlink?

**Comment by pescobar** [comment]
@boegel once this is merged all the dependencies for https://github.com/hpcugent/easybuild-easyconfigs/pull/4444 should be available

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2156.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b38739fc8f78ff86aa92bb919d447152 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2524.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7bd9332f83388772fafe5f52a3b58601 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**